tests/Bridge/Doctrine/Orm/Metadata/Property/DoctrineOrmPropertyMetadataFactoryTest.php

```diff
@@ -42,23 +42,6 @@ public function testCreateNoManager()
         $this->assertEquals($doctrineOrmPropertyMetadataFactory->create(Dummy::class, 'id'), $propertyMetadata);
     }
 
-    public function testCreateNoClassMetadata()
```

Comment:
Removed as this test was wrong. `ObjectManager::getMetadataFor` cannot return `null`.